packages/navstar/build

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -x
+
+source /opt/mesosphere/environment
+
+export CFLAGS="-I/opt/mesosphere/include -I/opt/mesosphere/active/libsodium/include"
+export LDFLAGS="-L/opt/mesosphere/lib -L/opt/mesosphere/active/libsodium/lib -Wl,-rpath=/opt/mesosphere/active/libsodium/lib"
+export CXXFLAGS=CFLAGS
+
+
+pushd /pkg/src/navstar
+./rebar3 update
+make rel
+popd
+
+cp -r /pkg/src/navstar/_build/prod/rel/navstar ${PKG_PATH}
+
+service=${PKG_PATH}/dcos.target.wants/dcos-navstar.service
+mkdir -p $(dirname $service)
+
+cat <<EOF > $service
+[Unit]
+Description=Navstar: A distributed systems & network overlay orchestration engine
+After=dcos-gen-resolvconf.service
+After=dcos-epmd.service
+
+[Service]
+Restart=always
+StartLimitInterval=0
+RestartSec=5
+WorkingDirectory=${PKG_PATH}/navstar
+EnvironmentFile=/opt/mesosphere/environment
+ExecStartPre=/bin/ping -c1 ready.spartan
+ExecStartPre=/usr/bin/env mkdir -p /var/lib/dcos/navstar/mnesia
+ExecStartPre=/usr/bin/env mkdir -p /var/lib/dcos/navstar/lashup
+ExecStart=${PKG_PATH}/navstar/bin/navstar-env foreground
+Environment=HOME=/opt/mesosphere
+User=dcos_navstar
+AmbientCapabilities=CAP_NET_ADMIN
```

Comment:
We can't use AmbientCapabilities because it is not available in older versions of systemd...